src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/ReflectionEmitMemberAccessor.cs

```diff
@@ -7,12 +7,27 @@
 using System.Diagnostics.CodeAnalysis;
 using System.Reflection;
 using System.Reflection.Emit;
+using System.Runtime.CompilerServices;
 
 namespace System.Text.Json.Serialization.Metadata
 {
     [RequiresDynamicCode(JsonSerializer.SerializationRequiresDynamicCodeMessage)]
     internal sealed class ReflectionEmitMemberAccessor : MemberAccessor
     {
+        private const int MaxInterpretedConstructorParameters = 64;
+        private static ReflectionMemberAccessor? s_reflectionMemberAccessor;
+        private readonly bool _isDynamicCodeInterpreted;
+
+        public ReflectionEmitMemberAccessor()
+        {
+#if !NETFRAMEWORK
```

Comment:
Shouldn't this be:

```suggestion
#if NETCOREAPP
```

? What happens for the netstandard build?

Reply:
The netstandard build doesn't use dynamic code, so the file isn't included at all. So in this case the two forms are equivalent. I used `NETFRAMEWORK` here to emphasize this.

Reply:
Fallback logic reverted.